Lib/fontmake/compatibility.py

```diff
@@ -16,11 +16,12 @@ def __exit__(self, type, value, traceback):
 
 
 class CompatibilityChecker:
-    def __init__(self, fonts):
+    def __init__(self, fonts, verbose=False):
         self.errors = []
         self.context = []
         self.okay = True
         self.fonts = fonts
+        self.verbose = verbose
```

Comment:
you don't want to store the verbose attribute in the instance. Loggers have a global state. You just need to import the top-level fontmake.logger and check if it's enabled for DEBUG

Reply:
the logger you want is actually fontmake.font_project.logger
https://github.com/googlefonts/fontmake/blob/0d48ff19b72dc0beed0c3835ea715392554e6dcf/Lib/fontmake/font_project.py#L46

Reply:
actually, sorry, the fontmake.compatibility module has its own logger at 
https://github.com/googlefonts/fontmake/blob/0d48ff19b72dc0beed0c3835ea715392554e6dcf/Lib/fontmake/compatibility.py#L3

use that one. It inherits the logging handlers of the parent, so if the parent is enabled for logging, and itself hasn't been tweaked specially, then it will also be verbose.

Reply:
Much nicer! thank you.